toolkit/themes/shared/design-system/config.js

```diff
@@ -0,0 +1,67 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* eslint-env node */
+
+const StyleDictionary = require("style-dictionary");
+const { fileHeader } = StyleDictionary.formatHelpers;
```

Comment:
```suggestion
```

Looks like we're not using this anymore so we can remove it (though I have a feeling we'll need to use it again down the line to add the MPL - a problem for another day 😅)

Reply:
Unfortunately I still need the file header since I have to do the formatting myself instead of being able to use "css/variables" from style dictionary.